po/README.md

```diff
@@ -0,0 +1,3 @@
+Please use [Crowdin](https://crowdin.com/project/xournalpp) to create or edit translations. 
+Should a language not be available, you can reach out to the developers by opening a new 
+issue or Gitter/Matrix (see links in main README).
```

Comment:
Looks like one of your rebases was not correct. This change is from a different PR and should not be part of your changes.

Reply:
I merged changes to `xournalpp:master` into my branch partway through work on my PR. I probably shouldn't have done this.

![image](https://user-images.githubusercontent.com/46334387/100514298-4f128a00-3128-11eb-8e6a-39ae3a3420a6.png)

These changes probably come from the commits labeled "**LittleHubba** authored and **personalizedrefrigerator committed**".

Reply:
I may have to fork `xournalpp:master` again and apply my changes to the fork... Should I do this and open a new PR?

Reply:
Normally you do not merge both branches but rebase on top of master.
You can use the git reflog to undo the merge and then do the rebase.

Reply:
Okay. Thank you for catching this!

Reply:
If using the reflog does not work, just cherry pick your changes to another branch locally and force push to your PR branch.

Reply:
Yes instead of using `git merge upstream:master` one should ever use `git rebase upstream:master` this will keep the history clean and you won't mess with foreign commits.